docs/tutorial.rst

```diff
@@ -804,33 +804,26 @@ in your file system:
 
     (psamm-env) $ psamm-model --model <PATH-TO-MODEL.YAML> fba
 
-By default, PSAMM fba will use the biomass function designated in the central
-model file as the objective function. If the biomass tag is not defined in a
-``model.yaml`` file or if you want to use a different reaction as the
-objective function, you can simply specify what the reaction ID is at the
-end of the command. For example to maximize the citrate synthase reactions,
-CS, the command would be as follows:
-
-.. code-block:: shell
-
-    (psamm-env) $ psamm-model fba CS
-
 The following is a sample of some output from the FBA command:
 
 .. code-block:: shell
 
     INFO: Model: Ecoli_core_model
-    INFO: Model Git version: 9812080
+    INFO: Model Git version: v0.1-819-g4974900-dirty
```

Comment:
You are getting a Git commit ID here from PSAMM itself because you didn't initialize a Git repository as explained earlier in the tutorial. I would just keep line identical to the original.